metron-interface/metron-alerts/src/app/pcap/pcap-filters/pcap-filters.component.spec.ts

```diff
@@ -179,19 +172,6 @@ describe('PcapFiltersComponent', () => {
     component.onSubmit();
   });
 
-  it('Port fields should be removed from request when set to empty', () => {
```

Comment:
Why was this test removed?

Reply:
It was intentional. We changed back the port type to string so this behavior not needed anymore.